runtime/legion/legion_ops.cc

```diff
@@ -670,6 +693,7 @@ namespace Legion {
       // Tell our parent context that we are done mapping
       // It's important that this is done before we mark that we
       // are executed to avoid race conditions
+      //fprintf(stderr, "track_parent: %d, %p\n", track_parent, this);
```

Comment:
Let's make sure we get rid of all these commented printfs before we do the final commit.